sdk-tools/src/sign.rs

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+//! Builder wrapper around SgxSign.
+
+use std::{
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+/// This structure wraps the execution of sgx_sign commands.
+#[derive(Default, Clone, Debug)]
+pub struct SgxSign {
+    /// The path to the sgx_sign executable.
+    sgx_sign_path: PathBuf,
+    /// Whether to ignore the presence of relocations in the enclave shared
+    /// object.
+    ignore_rel_error: bool,
```

Comment:
As someone not familiar with this code, it's hard to know what "rel" means in this context. Please consider renaming to "relocation."
```suggestion
    ignore_relocation_error: bool,
```